lib/yum.sh

```diff
@@ -28,6 +28,15 @@ yum_Q() {
   fi
 }
 
+yum_Qe() {
+  # in Centos8, repoquery takes 'reason' as format placeholder
+  centos_version="$(cat /etc/*elease | $GREP 'VERSION_ID=' | cut -d'"' -f2)"
```

Comment:
Can you pls try `$GREP 'VERSION_ID=' /etc/*elease`?

 It's also possible to avoid the use of `cut`, but it's also fine to use them. It's up to you ;)

Reply:
> It's also possible to avoid the use of cut, but it's also fine to use them. It's up to you ;)

Sure, it's done. Also I've fixed a swupd test failing [in previous commit](https://github.com/mondeja/pacapt/runs/3200357213?check_suite_focus=true).